config/locales/en/activemodel.yml

```diff
@@ -20,6 +20,10 @@ en:
           attributes:
             mentor_ids:
               blank: Select a mentor or not yet known
+        placements/add_placement_wizard/academic_year_step:
```

Comment:
You might also need this in `config/locales/en/placements/schools/placements.yml` between lines 13 - 22